src/lib/template.css

```diff
@@ -0,0 +1,104 @@
+:root {
```

Comment:
Question: if you are generating the CSS anyway, why do you need the template anymore?

Reply:
because I forgot to delete it 🤣